contracts/contracts/harvest/BaseHarvester.sol

```diff
@@ -284,19 +406,8 @@ abstract contract BaseHarvester is Governable {
     }
```

Comment:
We can also get rid of swapRewardToken, since it's governor only, and unlikely to be used.